src/main/java/org/jenkinsci/plugins/workflow/support/pickles/FilePathPickle.java

```diff
@@ -51,6 +51,9 @@ private FilePathPickle(FilePath v) {
             protected FilePath tryResolve() {
                 return FilePathUtils.find(slave, path);
             }
+            @Override protected FlowExecutionOwner getOwner() {
```

Comment:
What is this for?

Reply:
This was actually introduced for use from `PickleDynamicContext`, which was removed as of 5a7a0b29cad877d510074bc6c7d6f0a532214ce1, but it is independently useful—means that if a program _is_ still using `FilePathPickle` despite this PR + https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/86, and it is taking a while to resume, the build log will indicate why. Previously only `ExecutorPickle` implemented this. Generally pickles should implement this method if possible.